cmd/operator-sdk/main.go

```diff
@@ -15,51 +15,17 @@
 package main
 
 import (
+	"os"
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	// to ensure that `exec-entrypoint` and `run` can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
 	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/cli"
-	kbutil "github.com/operator-framework/operator-sdk/internal/util/kubebuilder"
-	"github.com/operator-framework/operator-sdk/internal/util/projutil"
-
-	log "github.com/sirupsen/logrus"
 )
 
 func main() {
-	// Use the new KB CLI when running inside a Kubebuilder project with an existing PROJECT file.
```

Comment:
Should we not still showing a msg: 

```
if is not show the info that is required to migrate the project when has not the Project file. See: https://github.com/operator-framework/operator-sdk/pull/3539/files#diff-fc95fb53b164f1e3dc64775883195e7bR28-R35


```

Reply:
I don't think so. SDK v1.0 has no knowledge of the legacy CLI/project layout.

Reply:
It has its pros and cons, I agree. But for alpha at least I think that is valid for we do not see many new issues raising asking about the same issue.